solutions/swb-reference/integration-tests/support/resources/environments/environments.ts

```diff
@@ -9,14 +9,24 @@ import CollectionResource from '../base/collectionResource';
 import Environment from './environment';
 
 export default class Environments extends CollectionResource {
-  public constructor(clientSession: ClientSession) {
+  private _projectId: string | undefined;
+
+  public constructor(clientSession: ClientSession, projectId?: string) {
```

Comment:
nit: I've been just passing the apiURL from the parent , that way if we add anything to the root It will keep adding up, we can address this after code freeze if needed

Reply:
I went with projectId because it was necessary for the `cleanup()` function for `Environment`. Can definitely revisit post freeze.